src/components/ui/InputField.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
actually in sign up I am viewing it as a base form with may fields . That's why tried to make it basic input field so that can handle it on that particular page.